plugins/rtmp-services/data/services.json

```diff
@@ -2128,6 +2128,8 @@
         },
         {
             "name": "nanoStream Cloud / bintu",
+		    "more_info_link": "https://www.nanocosmos.de/obs",
+		    "stream_key_link": "https://bintu-cloud-frontend.nanocosmos.de/organisation",
```

Comment:
There is inconsistent whitespace, please only use spaces (no tabs) in this file.